app/index.jsx

```diff
@@ -124,21 +125,25 @@ class App extends Component {
         const { settings, isConnectionOpened, symbol } = this.state;
 
         return (
-            <SmartChart
-                id={chartId}
-                symbol={symbol}
+            <SupportDetection
```

Comment:
as decouple detection code perhaps I should remove this. agree?

Reply:
yep